src/commands/stop.ts

```diff
@@ -0,0 +1,56 @@
+import chalk from 'chalk';
+import BaseCommand from "../base-command";
+import { DockerComposeUtils } from "../common/docker-compose";
+import inquirer from 'inquirer';
+import { DockerInspect } from '../common/docker-compose/template';
+
+export default class StopLocalDeployment extends BaseCommand {
```

Comment:
Based on how other commands are named we probably should just name this class `Stop` (or `StopCommand` if there happens to be some other `Stop` class)